scripts/bfasst/compare/waveform.py

```diff
@@ -79,8 +79,8 @@ def compare_netlists(self, design, run_waveform, tests, vivado):
 
         compare_waveforms.generate_files(multiple_files, paths, tests)
 
-        if compare_waveforms.run_test(paths):
-            Status(CompareStatus.SUCCESS)
+        if compare_waveforms.run_test(paths) is True:
```

Comment:
You shouldn't need to compare with True in an if statement.  Just use the result of the method invocation as the condition.
```suggestion
        if compare_waveforms.run_test(paths):
```